src/views/Home/components/Contributors/ProfileUpdated.tsx

```diff
@@ -10,11 +10,20 @@ interface Props {
 }
 
 const ProfileUpdated: FC<Props> = ({ date, type }) => {
+  const [formattedDate, setFormattedDate] = useState<string | null>(null);
   const labelText = type === ResourceType.CoreUnit ? 'Expense Report' : 'Profile Updated';
+
+  useEffect(() => {
+    if (date?.isValid) {
+      setFormattedDate(date?.toFormat('dd.MM.yyyy'));
+    } else {
+      setFormattedDate('No data');
+    }
+  }, [date]);
   return (
     <Container>
       <Label>{labelText}</Label>
-      <DateContainer>{date?.toFormat('dd.MM.yyyy') ?? 'No data'}</DateContainer>
+      <DateContainer>{formattedDate}</DateContainer>
```

Comment:
we don't need a side effect for this case. it can be done inline:

```
date?.isValid ? date.toUTC().toFormat(<format>) : 'No data'
```

Reply:
done